prog/vm/github_runner.rb

```diff
@@ -260,6 +260,17 @@ def before_run
 
     if vm
       vm.private_subnets.each { _1.incr_destroy }
+      if github_runner.workflow_job&.fetch("conclusion") == "failure"
```

Comment:
Can we save logs even if the runner doesn't have a workflow job `github_runner.workflow_job`? We'd like to debug recycled runners more effectively

Reply:
I have updated it, also removed the nap 15 change since we run the commands anyway when it is nil.